packages/block-editor/src/store/defaults.js

```diff
@@ -156,6 +156,9 @@ export const SETTINGS_DEFAULTS = {
 	// Allows to disable block locking interface.
 	canLockBlocks: true,
 
+	// Allows to disable Openverse media category in the inserter.
+	openverseMediaCategory: true,
+
```

Comment:
Suggestion: Maybe we should indicate that option is a boolean in the name? We can use `enable` or `supports` prefixes. What do you think?

Reply:
I had this named `enableOpenverseMediaCategory` and was looking at the other boolean ones that didn't contain it... Hm.. maybe it's better to update to `enableOpenverseMediaCategory`.

Reply:
Yeah. There are a few old settings that don't follow this convention. But most of the new settings' names indicate boolean values.